src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -78,4 +78,9 @@ public GuiSettings getGuiSettings() {
     public void setGuiSettings(GuiSettings guiSettings) {
         model.setGuiSettings(guiSettings);
     }
+
+    @Override
+    public double calculateTotalTransaction(ObservableList<Client> filteredClientList) {
+        return model.calculateTotalTransaction(filteredClientList);
```

Comment:
Is it possible to add a test to cover this line?

Reply:
I realized my test for this line was private instead of public. It should be covered now